docs/index.md

```diff
@@ -66,3 +66,23 @@ tests.
 
 Example of creating and deploying a static website to AWS. Terraform is used to create infrastructure. S3 is used to 
 host the website. Route 53 is used for DNS resolution. CloudFront and ACM are used for SSL.
+
+## Get Involved
+
+### Connect with the DevOps Tooling
+
+The DevOps Tooling Working Group is a group of individuals from different teams working together on the best practices 
+of using DevOps at BYU.
+
+- GitHub team: [DevOps Tooling](https://github.com/orgs/byu-oit/teams/devops-tooling)
+- Slack channel: [#devops-tools](https://byu-oit.slack.com/archives/CQ2BE663T)
```

Comment:
Maybe also mention the devops-tools-comms channel?

Reply:
Or official channels, like contacting our new Product Manager?

This might be worth separating into its own page.

Reply:
Fixed